d2game/d2player/key_to_string.go

```diff
@@ -0,0 +1,170 @@
+package d2player
+
+import (
+	"github.com/OpenDiablo2/OpenDiablo2/d2common/d2enum"
+	"github.com/OpenDiablo2/OpenDiablo2/d2core/d2asset"
+)
+
+// KeyToString returns a string representing the key
+func KeyToString(k d2enum.Key, assetManager *d2asset.AssetManager) string {
+	switch k {
+	case -1:
+		return assetManager.TranslateString("KeyNone")
+	case d2enum.KeyTilde:
```

Comment:
this can be a method of `d2enum.Key` like this:
```golang
func (k Key) String() string {
    // switch statement here
}
```

Reply:
I originally did that, but as we need the AssetManager I figured it shouldn't be associated to the `Key` type